macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/VPNAppEventsHandler.swift

```diff
@@ -104,12 +104,16 @@ final class VPNAppEventsHandler {
     ///
     @MainActor
     private func loginItemsControlCheckpoint(canRestart: Bool, loginItemsManager: LoginItemsManaging) async {
+        guard loginItemsManager.isAnyEnabled(LoginItemsManager.vpnLoginItems) else {
+            return
+        }
```

Comment:
This is the first part of the fix: we can only control through IPC a login item that's enabled.  We can't control a login item that a user disabled.

Specifically, we replaced `isAnyInstalled` in the `false` case below.